### ISSUE TITLE
[Question] when I open the Camera at the begining, The display will be briefly enlarged（in ios 17）

### ISSUE THREAD

**Issue by Dgkiller-zyx** [mebjas/html5-qrcode#851]
when I open the Camera at the begining, The display will be briefly enlarged（in ios 17）
It does not occur in versions below ios 17, who can tell me why,and how to solve this problem

https://github.com/mebjas/html5-qrcode/assets/122333837/0e51f69a-2396-4855-8bc9-5109b8eb40c9

**Comment by ROBERT-MCDOWELL** [comment]
THIS HAS NOTHING TO DO WITH HTML5-QRCODE BUT WITH YOUR CAMERA SETTINGS AT YOUR OS AND BROWSER LEVEL

**Comment by Dgkiller-zyx** [comment]
> THIS HAS NOTHING TO DO WITH HTML5-QRCODE BUT WITH YOUR CAMERA SETTINGS AT YOUR OS AND BROWSER LEVEL

why, below ios 17 not happend, the same settings

**Comment by ROBERT-MCDOWELL** [comment]
you are just saying it's a OS difference! it has NOTHING TO DO WITH JAVASCRIPT CODING

**Comment by Dgkiller-zyx** [comment]
> you are just saying it's a OS difference! it has NOTHING TO DO WITH JAVASCRIPT CODING

so, the ios will verup again and again, but the htlm5-qrcode not  compatible with iOS17，maybe it has a solution，but I don't know，so I asked this question. You say is an issue with OS and Browser. should I ask the developers of Safari Browser and iOS???

**Comment by ROBERT-MCDOWELL** [comment]
YES! html5-qrocde is a coded in JAVASCRIPT which has its owen SANDBOX, meaning that every JS engine inside a browser or app DEPEND from the OS
